### DISCUSSION TITLE
Self introduction

### DISCUSSION THREAD

**Discussion by muvalika** [wagtail/gsoc#118] [General]
Hello Wagtail,
 My name is Muvalika Gr, and I'm currently pursuing a bachelor's degree at MBU, Andhra Pradesh. I'm excited about the opportunity to contribute my coding skills, particularly in GUI tools, to the  project 1 at Wagtail. I've been following the project's progress with great interest, and I believe my skills and eagerness to learn can be valuable assets to the team. I'm particularly interested in bettermenting with distinct's in project-1 . Thank you for your time and consideration. I'm eager to engage with Wagtail. 
Thank you 
 Regards
 Muvalika Gr

**Comment by iqraakhtar09** [comment]
Hi, I’m Iqra I'm a student of CS at Virtual University of Pakistan. Well, this is the first time I’m very excited and feeling full of potential to contribute to GSOC. I’m also starting to see past repositories and trying to contribute to those as well, Please advise me on some tips and tricks so that i can crack this organization for GSOC.
you can check my LinkedIn
https://www.linkedin.com/in/iqra-akhtar-/

**Comment by muvalika** [comment]
-

   Hello Iqra, hope this message finds you well.
   -


   -

   In your proposal mention your previous projects and open source
   contributions if any
   -

   After you choose the project make sure to explain yourself regarding why
   you're the right fit for that project
   -

   More than the proposal letter the contributions you make and the
   relationship with your mentor matters
   -

   Open the GitHub links of those projects and try to solve the easy issues
   and then move on to the complex ones
   -
      - 👏
      - 👍
      - 👎
      - 😊
      - 😞
      -

   Make sure to contribute to more than one project


Best Regards
Muvalika Gr

**Comment by iqraakhtar09** [comment]
@muvalika Thanks for your guidelines. I have a question: how can I mention my entire project in my proposal? Can I just mention my linkdin and, in purpose, just tell few lines about my projects?

**Comment by iqraakhtar09** [comment]
@muvalika Thanks for your guidelines. I have a question: how can I mention my entire project in my proposal? Can I just mention my linkdin and, in purpose, just tell a few lines about my projects?

**Comment by muvalika** [comment]
Hello Iqra,
hope this message finds you well.
I would like to suggest a brief description of how I would like to
implement the project further.
The description shall not be the same as given in the original website, and
very foremost apart from the proposal it mainly depends on how close you
share the suggestions of improvement with your mentors.


Regards
Muvalika Gr

**Comment by Valley-15** [comment]
Greetings,

I hope this message finds you in good spirits! I came across your project on GitHub and was truly impressed by what you're doing. My name is Adnaan, and I'm currently pursuing my BTech in Computer Science and Engineering at VNR Vignana Jyothi Institute of Engineering and Technology in Hyderabad.

I'm reaching out because I have a sincere interest in contributing to open-source projects like yours. I'm enthusiastic about expanding my knowledge and skills in the field, and I believe that collaborating on projects such as yours will offer me valuable insights and experiences.

Your project particularly resonates with me because I'm deeply passionate about initiatives that make a meaningful impact. It seems like your project aligns perfectly with my interests and aspirations. I'm eager to dive in, contribute in any way I can, and soak up as much knowledge as possible along the journey.

If you could provide any guidance on how to get started, I would greatly appreciate it. I'm truly excited about the prospect of being part of your project and contributing to its success.

Looking forward to hearing from you!

Best regards,
Adnaan

**Comment by muvalika** [comment]
Hello Adnaan,
Pleasure hearing about you.
Well I'm not a mentor, I am 1 st year undergraduate from Mohan Babu
University, Andhra Pradesh.
Looking into the message I would like to suggest that you mention "what's
that special about your thought to  choose you instead of others".


Best Regards
Muvalika Gr

**Reply by Valley-15** [reply]
Thanks, Muvalika. Can you please guide me on how to get started? I have some knowledge about web development, but I feel a bit overwhelmed when I see a huge codebase. I'm also unsure where to start. Any advice or tips you could provide would be greatly appreciated.

**Comment by muvalika** [comment]
Based on the Summer internships:
1. Gsoc(google summer of code)
2. Outreachy
3. Moss
4. Mlh
5. Sih
practicing through leetcode, hacker rank, and many such web sites.
I would foremost suggest you to go through udemy and coursera
certifications.

**Reply by Valley-15** [reply]
Thanks Muvalika. I'm very impressed by your skills. Which organization are you participating with for GSoC 24?

**Reply by Valley-15** [reply]
Muvalika, did you participate in all of these:

1. GSoC (Google Summer of Code)
2. Outreachy
3. MOSS
4. MLH
5. SIH

Let me know!

**Comment by muvalika** [comment]
I have applied for each above, as I am year 1 student so this would be my
first year participating.

**Reply by Valley-15** [reply]
Muvalika could you please explain me what is MOSS also provide links of it if possible.
By the  way I am happy that you are first year student but are very enthusiastically participating in all programs and not relying only on book knowledge.
Happy coding!

**Comment by muvalika** [comment]
https://youtube.com/shorts/T55PXmsD9mo?si=8Lkz-n4UcEkYQdo5




You can have this video as a reference.

Could you please help me knowing where's the link for wagtail's proposal
submission.


Best Regards
Muvalika GR

**Comment by ObMTHsTech** [comment]
Hi Everyone, I’m excited to be here.
I’m a final year mathematics student with background and on hands experience in software development and data engineering. I currently volunteer as Data Analyst for iMasons - A global nonprofit for digital infrastructure industry professionals.
I’m here for the Google Summer of Codes (GSoC) Contributor with Wagtail.
It’s my first experience with Django. I just went through the tutorials and programming with wagtail is as thrilling as a game. It’s less complex than where we came from (html, css, JavaScript, C, …). I hope to make something beautiful out of it.
I’m looking forward to doing amazing things with Wagtail together with my wonderful mentors and contributors.
Above coding, my skills and experience include Project Management, Coaching, Business Development and Research.
I’ll be willing to give a helping hand to any contributor/student with difficulty (even when I’m a student myself :face_palm::skin-tone-5:).
Let’s stay in touch!

Regards, 
Obongofon Udombat

**Comment by JaiswalShivang** [comment]
Hi everyone , My self Shivang Jaiswal i am currently pursuing my bachelor degree in computer science at National Institute of Technology Patna
